### ISSUE TITLE
🔧  Reference to property 'delegate' in closure requires explicit use of 'self' to make capture semantics explicit

### ISSUE THREAD

**Issue by Romick2005** [cuvent/react-native-vision-camera#2071]
### How were you trying to build the app?

![image](https://github.com/mrousavy/react-native-vision-camera/assets/17779660/082e7611-bad6-46cf-9c94-e5da8a1d8b27)

### Full build logs

```tsx
Reference to property 'delegate' in closure requires explicit use of 'self' to make capture semantics explicit

PS Just try to build iOS and you will see the issue that could be fixed by adding 'self'
```

### Project dependencies

```json
Latest
```

### VisionCamera Version

3.6.0

### Target platforms

iOS

### Operating system

MacOS

### Can you build the VisionCamera Example app?

I didn't try (⚠️ your issue might get ignored & closed if you don't try this)

### Additional information

- [ ] I am using Expo
- [ ] I have enabled Frame Processors (react-native-worklets-core)
- [X] I have read the [Troubleshooting Guide](https://react-native-vision-camera.com/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

**Comment by mrousavy** [comment]
My bad, fixed on main

**Comment by mrousavy** [comment]
released in 3.6.1
